WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -97,11 +99,14 @@ - (XCElementSnapshot *)fb_parentMatchingType:(XCUIElementType)type
   return snapshot;
 }
 
-- (XCElementSnapshot *)fb_parentMatchingOneOfTypes:(NSArray<NSNumber *> *)types
+- (XCElementSnapshot *)fb_findVisibleParentMatchingOneOfTypesAndHasMoreThanOneChild:(NSArray<NSNumber *> *)types
```

Comment:
ok. I am not familiar with filtering. I work on obj-c for only `WDA`. so here `filter` here will be logic from `doesSnapshotHasMoreThanOneVisibleChildSnapshot`?